docs/sources/alerting/set-up/_index.md

```diff
@@ -1,15 +1,61 @@
 ---
 aliases:
   - unified-alerting/set-up/
-description: How to set up additional alerting features and integrations
-title: Additional setup
-weight: 600
+labels:
+  products:
+    - oss
+description: How to set up alerting features and integrations
+title: Set up Alerting
+weight: 107
 ---
 
-# Additional setup
+# Set up Alerting
 
-Alerting supports a plethora of configurations, from [configuring external Alertmanagers]({{< relref "./configure-alertmanager" >}}) to routing Grafana Managed Alerts outside of Grafana, to defining your alerting setup [as-code using Provisioning]({{< relref "./provision-alerting-resources" >}}).
+Configure the features and integrations that you need to create and manage your alerts.
 
-Setup Grafana Alerting high-availability mode by [following this guide]({{< relref "./configure-high-availability" >}}).
+**Note:**
 
-Connect your alerting setup to Grafana OnCall by [following this guide](/docs/oncall/latest/integrations/available-integrations/add-grafana-alerting/).
+These are set up instructions for Grafana Alerting Open Source.
+
+To set up Grafana Alerting for Cloud, see
+
+## Before you begin
+
+- Configure your [data sources]({{< relref "../../administration/data-source-management" >}})
+- Check which data sources are compatible with and supported by [Grafana Alerting]({{< relref "../fundamentals/data-source-alerting" >}})
+
+## Set up Alerting
+
+To set up Alerting, you need to:
+
+1. Configure alert rules
+
+   - Create Grafana-managed or Mimir/Loki-managed alert rules and recording rules
+
+1. Configure contact points
+
+   - Check the default contact point and update the email address
+
+   - [Optional] Add new contact points and integrations
+
+1. Configure notification policies
+
+   - Check the default notification policy
+
+   - [Optional] Add additional nested policies
+
+   - [Optional] Add labels and label matchers to control alert routing
+
+1. [Optional] Integrate with [Grafana OnCall]
+   (/docs/oncall/latest/integrations/grafana-alerting)
```

Comment:
This link doesn't seem to render for me, intentional?

Reply:
fixed